.travis.yml

```diff
@@ -85,8 +80,8 @@ script:
 
 after_success:
   - if [[ $COVERALLS == 'true' ]]; then coveralls; fi
-  - |
-     test ${TRAVIS_PULL_REQUEST} == "false" && \
-     test ${TRAVIS_BRANCH} == ${GH_DOC_BRANCH} && \
-     test "${BUILD_DOCS}" == "true" && \
-     bash ${TRAVIS_BUILD_DIR}/maintainer/deploy_docs.sh
+  - if test ${TRAVIS_PULL_REQUEST} == "false" && \
+       test ${TRAVIS_BRANCH} == ${GH_DOC_BRANCH} && \
+       test "${BUILD_DOCS}" == "true"; then
+           bash ${TRAVIS_BUILD_DIR}/maintainer/deploy_docs.sh
+    if
```

Comment:
Hmm, I've fixed that already a couple of hours ago, but apparently pushed to the wrong branch...